src/controllers/request-controller.ts

```diff
@@ -124,7 +125,12 @@ export class RequestController {
 
       // request was newly created
       if (body) {
-        Metrics.count(METRIC_NAMES.CTRL_NEW_ANCHOR_REQUEST, 1, { source: parseOrigin(req) })
+        const v3Params = requestParams as RequestAnchorParamsV3
+        Metrics.count(METRIC_NAMES.CTRL_NEW_ANCHOR_REQUEST, 1, {
+          source: parseOrigin(req),
+          jsCeramicVersion: v3Params.jsCeramicVersion,
```

Comment:
@gvelez17 is this sufficient to get what we want to track what version people are running based on the requests coming into CAS?

Reply:
Should we also add this to `getStatusForCid` for status requests?

Reply:
We certainly could, but seems less useful and there is *some* cost in bandwidth to sending this on every request.

Reply:
I figured how many requests are created by which versions was a more useful metric than how much polling is happening from which version

Reply:
it should be - we don't really want more than 2 version parameters per node, I would think.  these are the key ones.  If something else changes we can try to enforce bumping the minor version of js-ceramic to recognize it